core/templates/pages/exploration-player-page/services/audio-translation-language.service.ts

```diff
@@ -42,10 +42,10 @@ export class AudioTranslationLanguageService {
     private browserCheckerService: BrowserCheckerService,
     private languageUtilService: LanguageUtilService) {}
 
-  _currentAudioLanguageCode: string = null;
-  _allAudioLanguageCodesInExploration: string[] = null;
-  _explorationLanguageCode: string = null;
-  _automaticTextToSpeechEnabled: boolean = null;
+  _currentAudioLanguageCode: string | null = null;
+  _allAudioLanguageCodesInExploration: string[] = [];
+  _explorationLanguageCode: string = '';
```

Comment:
This should be `null` too, for the same reason as before.

Reply:
I checked all the times `explorationLanguageCode` was being set and it is never `null`.

Reply:
I do believe `null` would not be needed as its value, should I still put `null` in the type?

Reply:
Wait, then why this variable needs to be set?

Reply:
with reference to what I said before, I checked *all* the calls to `init` and the value of `explorationLanguageCode` is never set to `null` 
https://github.com/oppia/oppia/blob/cc565de098c3fba95355fa01db40769b36c4d4aa/core/templates/pages/exploration-player-page/services/audio-translation-language.service.ts#L139-L147

then I checked whether `AudioTranslationLanguageService` is being called without `init` and it was, this was a bad mistake from my end.